src/plugins/notebook/components/Notebook.vue

```diff
@@ -226,13 +226,13 @@ export default {
         createNotebookStorageObject() {
             const notebookMeta = {
                 name: this.internalDomainObject.name,
```

Comment:
This is going to go stale. I understand why it's needed (to populate the menu), but I can guarantee it's going to go stale. A user at some point is going to rename their notebook. The notebook menu is going to have to resolve the default notebook asynchronously, and only add the default notebook entry once it has resolved it. It could even do this on startup (using `openmct.on('start', () => ...)`) to get ahead of the user.

Reply:
to solve this keeping name property in notebook storage and checking/updating it when notebook changes.
I think `(using openmct.on('start', () => ...))` may not work as user can change default notebook and/or rename anytime.

Reply:
The second option. Track the ID of the default notebook and pull it from the database when needed. Maybe you could use [onChanged](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/onChanged) to update dynamically when the default notebook changes due to a user action.

You will need to resolve the default notebook on load as well. My suggestion with `.on('start'` was to avoid the delay in resolving an object asynchronously when the user clicks on the menu. You could also probably just do it from the menu component itself. In any case, do it before the user clicks on the menu so there's no pause with nothing happening.

Reply:
Added `getDefaultNotebookObject` on menu component to resolve the default notebook on load